typespec/Applications.Core/containers.tsp

```diff
@@ -322,6 +322,9 @@ enum VolumePermission {
 enum IAMKind {
   @doc("Azure Active Directory")
   azure,
+
+  @doc("Amazon Web Services IAM")
+  aws,
```

Comment:
Enum with one value didn't create the `Possible...Enum...Values` function. I added this to test that out. We can change this.

Reply:
Is this supposed to return an enum function? Functions would only be generated for schema designated as @action